cmd/bbolt/command_surgery_cobra.go

```diff
@@ -78,3 +78,39 @@ func surgeryClearPageElementFunc(cmd *cobra.Command, args []string) error {
 	fmt.Fprintf(os.Stdout, "All elements in [%d, %d) in page %d were cleared\n", surgeryStartElementIdx, surgeryEndElementIdx, surgeryPageId)
 	return nil
 }
+
+func newSurgeryAbandonFreelistCommand() *cobra.Command {
+	abandonFreelistCmd := &cobra.Command{
+		Use:   "abandon-freelist <bbolt-file> [options]",
+		Short: "abandon the freelist from both meta pages",
```

Comment:
Nit - Capital "A" to start the sentence and stay consistent with the existing command.

Reply:
updated & thx